src/jitlayers.h

```diff
@@ -174,9 +230,9 @@ class JuliaOJIT {
     CompileLayerT CompileLayer;
     SymbolTableT GlobalSymbolTable;
     SymbolTableT LocalSymbolTable;
+    DenseMap<void*, StringRef> ReverseLocalSymbolTable;
 };
 extern JuliaOJIT *jl_ExecutionEngine;
-JL_DLLEXPORT extern LLVMContext jl_LLVMContext;
```

Comment:
Why unexport this? CUDAnative needs access to the Julia context. Can't get it from the module (`Value::getContext` doesn't seem wrapped in the C API), also because we need it _before_ getting the module (eg. to perform `llvmcall`s with LLVM.jl).

Reply:
Cxx needs it as well.